docs/usage.md

```diff
@@ -92,28 +98,28 @@ Magmap can handle several types of input that can be used for different purpose.
 The indexes input is used by Sourmash to select genomes that can be downloaded in a second step and added to the pipeline.
 It is provided with the `--indexes` parameter and it is a path (local or remote).
 
-```
-https://github.com/nf-core/test-datasets/raw/magmap/testdata/sourmash_test.index.sbt.zip
+```bash
+nextflow run nf-core/magmap -profile docker --outdir results/ --input samples.csv --genomeinfo localgenomes.csv --indexes 'https://github.com/nf-core/test-datasets/raw/magmap/testdata/sourmash_test.index.sbt.zip'
 ```
 
 N.B.: the sbt files can be generated with sourmash (check it [here](https://sourmash.readthedocs.io/en/latest/index.html)) or some can be found in sourmash documentation website.
 E.g. "sourmash gtdb sbt" you will find the indexes for gtdb.
 
 #### Metadata input
 
-Magmap accepts several metadata as `.csv` files that provides information about the genomes that you will use in the pipeline.
+Magmap accepts several metadata as `.csv` files that provides information about the genomes that you will use in the pipeline. Each metadata can be used to get different information about you genomes as their quality (CheckM) or the taxonomy (gtdb). Magmap will merge all these tables and create a new one: each row will correspond to a genome (based on its accno) followed by several columns.
```

Comment:
```suggestion
Magmap accepts several types of metadata as `.csv` files that provides information about the genomes that you will use in the pipeline. Each type of metadata can be used to get different information about you genomes as their quality (CheckM) or the taxonomy (gtdb). Magmap will merge all these tables and create a new one: each row will correspond to a genome (based on its accno) followed by several columns.
```

Reply:
Actually, we're only accepting taxonomy and genome statistics, so this text gets unnecessarily abstract I think. Or maybe, we don't need this part at all and can keep the header (or change to "Genome metadata input"?) and only describe each particular type below?

Reply:
I changed the title and part of the text.